### DISCUSSION TITLE
WTF学院会计表

### DISCUSSION THREAD

**Discussion by AmazingAng** [AmazingAng/WTF-Solidity#113] [General]
WTF Academy所有收入/支出均透明，以$PEOPLE计价，且公开于这个帖子中。我们希望能够在2022年底达到收支平衡。

收支也记录在公开在这个google sheet里：https://docs.google.com/spreadsheets/d/1V1uuHCG5u1vhS7upTs-s7QdMOmfKL8qR9OYFebHwGu4/edit?usp=sharing

**Comment by 0xmi** [comment]
。

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: 0xAA ***@***.***>
Sent: Saturday, July 23, 2022 2:52:11 PM
To: AmazingAng/WTFSolidity ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [AmazingAng/WTFSolidity] WTF学院会计表 (Discussion #113)


WTF Academy所有收入/支出均透明，以$PEOPLE计价，且公开于这个帖子中。我们希望能够在2022年底达到收支平衡。

收支也记录在公开在这个google sheet里：https://docs.google.com/spreadsheets/d/1V1uuHCG5u1vhS7upTs-s7QdMOmfKL8qR9OYFebHwGu4/edit?usp=sharing

日期      会计类别    收入/支出   描述      链接      数额 ($PEOPLE)
2022.06.05      教程      支出      WTF Solidity第一次审核       #18<https://github.com/AmazingAng/WTFSolidity/discussions/18>   -10000
2022.07.04      社群      支出      WTF社群6月奖励       https://twitter.com/0xAA_Science/status/1543689933713391617?s=20&t=logZSoDsnHIRnIDhBlsisQ       -10000
2022.07.18      捐款      收入      Gitcoin GR14捐款  https://gitcoin.co/grants/6090/wtf-solidity     120000
社群资产                                    100000

―

**Comment by AmazingAng** [comment]
4月 WTF Academy 支出

1. Hongkong Web3 Festival（5人食宿）：2500u
2. 泰国出差 Pagoda + SEABW 东南亚区块链周（0xAA + Tank2人食宿）：1500u
3. 域名购买（git.id 和 gitid.eth），用于未来社区项目: 2000u

总计 6000u ，由0xAA垫付。
